apps/order/schema.py

```diff
@@ -96,6 +98,79 @@ class Meta:
         filterset_class = OrderFilterSet
 
 
+class OrderTypeForStatistic(DjangoObjectType):
+    total_quantity = graphene.Int()
+
+    class Meta:
+        model = Order
+        fields = ('id', 'order_code', 'total_price')
+
+    def resolve_total_quantity(root, info, **kwargs):
+        return root.book_order.aggregate(Sum('quantity'))['quantity__sum']
+
+
+stat_to = datetime.date.today()
+stat_from = stat_from = stat_to - datetime.timedelta(30)
+
+
+class OrderStatType(graphene.ObjectType):
+    total_quantity = graphene.Int()
+    books_uploaded_count = graphene.Int()
+    orders_completed_count = graphene.Int()
+    books_ordered_count = graphene.Int()
```

Comment:
Let's add a description specifying the last 3 months using `description` argument.

Reply:
@thenav56  Description added